api/applicant/models/applicant.settings.json

```diff
@@ -64,6 +64,15 @@
       "type": "component",
       "repeatable": true,
       "component": "skills.skills"
+    },
+    "discordUsername": {
+      "type": "string"
+    },
+    "extraInfo": {
```

Comment:
What kind of extra info?

Reply:
This applicant collection is basically for volunteers to apply for an open position at a product. We have form fields like full name, age, skills etc. This one is to give applicant Extra space for them to write questions, concerns or more info about why they should be selected. The field name in the UI is "do you have anything else you want to share with us?"